src/main/java/frc/robot/commands/auto/PathPlannerCommand.java

```diff
@@ -50,20 +48,20 @@ public PathPlannerCommand(List<PathPlannerTrajectory> pathGroup, int pathIndex,
         if (pathIndex < 0 || pathIndex > pathGroup.size() - 1){
             throw new IndexOutOfBoundsException("Path index out of range"); 
         } 
-        PathPlannerTrajectory path = pathGroup.get(pathIndex);
-    
+        PathPlannerTrajectory path = PathPlannerTrajectory.transformTrajectoryForAlliance(pathGroup.get(pathIndex),
+            DriverStation.getAlliance());
+
         addCommands(
             (pathIndex == 0 && resetPose ? new InstantCommand(() -> {drive.setPigeonYaw(path); drive.resetOdometry(path.getInitialHolonomicPose());}) : new DoNothing()),
             new PrintCommand("Number of paths: " + pathGroup.size()),
             new PPSwerveControllerCommand(
                 path, 
                 drive::getPose, // Pose supplier
-                DriveConstants.kKinematics, // SwerveDriveKinematics
                 drive.getPathplannerXController(), // X controller can't normal PID as pathplanner has Feed Forward 
                 drive.getPathplannerYController(), // Y controller can't normal PID as pathplanner has Feed Forward 
                 drive.getPathplannerRotationController(), // Rotation controller can't normal PID as pathplanner has Feed Forward 
-                drive::setModuleStates, // Module states consumer
-                true,
+                (chassisSpeeds) -> { drive.setChassisSpeeds(chassisSpeeds, false); }, // chassis Speeds consumer
+                false,  // use Alliance color
```

Comment:
remove the path preseting and revert the use aliance color to true or else it will brake auto paths in main if we switch teams between enables.